charts/opentelemetry-kube-stack/values.schema.json

```diff
@@ -1606,27 +1605,7 @@
           "type": "string"
         },
         "image": {
-          "description": "Image use in both standalone and agent configs",
```

Comment:
I think the correct fix should be to follow the scheme registry/tepository:tag design and update related tpl, not change it to string.